content/docs/command-reference/exp/run.md

```diff
@@ -0,0 +1,108 @@
+# exp run
+
+Run an experiment: reproduce a variation of a committed
+[pipeline](/doc/command-reference/dag) in a hidden project branch.
+
+> Similar to `dvc repro` but for
+> [experimentation](/doc/user-guide/experiment-management).
+
+## Synopsis
+
+```usage
+usage: dvc exp run [-h] [-q | -v] [-f]
+                   [<repro_options> ...]
+                   [--params [<filename>:]<params_list>]
+                   [-n <name>] [--queue] [--run-all] [-j <number>]
+                   [targets [targets ...]]
+
+positional arguments:
+  targets               Stages to reproduce. 'dvc.yaml' by default.
+```
+
+## Description
+
+Provides a way to execute and track `dvc experiments` in your
+<abbr>project</abbr> without polluting it with unnecessary commits, branches,
+directories, etc.
+
+> `dvc exp run` is equivalent to `dvc repro` for <abbr>experiments</abbr>. It
+> has the same behavior when it comes to stage execution (restores the
+> dependency graph, etc.). It differs from in that experiment dependencies and
+> outputs will always be <abbr>cached</abbr> (so they can be restored later).
```

Comment:
I don't understand this sentence. Are `dvc repro` dependencies and outputs not cached?

Reply:
Oops "from" wasn't needed there, typo. "It differs in that dependencies and outputs will always be cached". But yeah wait... I need to re-think that (taken [from Peter's original PR](https://github.com/iterative/dvc.org/pull/1926/files#diff-162526d7eb1876fbb25840504db2c51860ae3c1b7af5b5f6431b0828c827189fR35-R36))... ⌛

Reply:
I think this refers to the fact that there's no `--no-commit` option, like in `repro`. But that's already mentioned in the **Options** section so I'll remove it from here 👍